main.py

```diff
@@ -22,11 +22,23 @@
 import flask_login as login
 from flask_admin.contrib import sqla
 from flask_admin import helpers, expose, BaseView
-from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_mail import Mail,Message
 
 app = utilities.init_flask(__name__)
+app.config.from_object(__name__)
+app.config.update(dict(
+    MAIL_SERVER='smtp.gmail.com',
+    MAIL_PORT=465,
+    MAIL_USE_TLS = False,
+    MAIL_USE_SSL= True,
+    MAIL_USERNAME = 'anywaytesttest@gmail.com',
```

Comment:
Anyway@anyway.co.il is functional



‫ב-19 במאי 2015, בשעה 09:04, ‏‏Daniel Hershcovich ‏<notifications@github.com> כתב/ה:‬

> In main.py:
> 
> >  
> >  app = utilities.init_flask(__name__)
> > +app.config.from_object(__name__)
> > +app.config.update(dict(
> > +    MAIL_SERVER='smtp.gmail.com',
> > +    MAIL_PORT=465,
> > +    MAIL_USE_TLS = False,
> > +    MAIL_USE_SSL= True,
> > +    MAIL_USERNAME = 'anywaytesttest@gmail.com',
> Is this account functional?
> Will you replace it with the anyway@anyway.co.il address from @galraij ?
> 
> —
> Reply to this email directly or view it on GitHub.
>